lib/TPP/IR/MatcherUtils.cpp

```diff
@@ -151,7 +144,82 @@ static bool hasReluBody(Operation *op, SmallVectorImpl<Value> *captured) {
     }
     return false;
   };
-  return (getOperand(maxfLhs, maxfRhs) || getOperand(maxfRhs, maxfLhs));
+
+  // Multiple patterns map to Relu.
+  if (isa<arith::MaximumFOp>(innerOp)) {
```

Comment:
I would change as: `if (auto maxfOp = dyn_cast<arith::MaximumFOp>(innerOp))`. This allows you yo avoid the cast in line 153.